bucketlist_app/context_processors.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+
+def set_environment(request):
+    return {
+        'environment': os.getenv('NODE_ENV') if os.getenv('NODE_ENV') else 'development'
+    }
```

Comment:
Why is this used again here?

Reply:
I'm using it because there are some scripts dependent on the environment. Check this out: https://github.com/BolajiOlajide/BucketListApplication/blob/develop/bucketlist_app/templates/base.html#L11

The minimized css and js are used in production but not in `development` environment. I should change the variable name though since there isn't a `node` application here.